### ISSUE TITLE
LBQuestions not work after reinstall

### ISSUE THREAD

**Issue by heaMan22** [Dhanus3133/Leetbuddy.nvim#23]
After reinstalling the plugin, :LBQuestions does not work, and the input will report an error:

```
E5108: Error executing lua: ...are/nvim/lazy/LeetBuddy.nvim/lua/leetbuddy/questions.lua:129: attempt to index local 'problem' (a nil value)
stack traceback:
	...are/nvim/lazy/LeetBuddy.nvim/lua/leetbuddy/questions.lua:129: in function 'key_func'
	...hare/nvim/lazy/telescope.nvim/lua/telescope/mappings.lua:257: in function <...hare/nvim/lazy/telescope.nvim/lua/telescope/mappings.lua:256>
```

**Comment by Dhanus3133** [comment]
You have clicked enter too early before even the problem is displayed. Try selecting the problem after it is displayed.

**Comment by heaMan22** [comment]
1.  logged in leetcode.cn
2.  input :LBQuestions
3.  however,  no problem is displayed in telescope window

finally, the problem solved by resetting the latest commit(98d8e5d) to d73fa0a
```
$ cd ~/.local/share/nvim/lazy/LeetBuddy.nvim
$ git reset --hard d73fa0a
```

**Comment by Dhanus3133** [comment]
Fixed the issue!
